microservices/s3_to_redshift/gov_assets.json

```diff
@@ -0,0 +1,65 @@
+{
+    "bucket": "sp-ca-bc-gov-131565110619-12-microservices",
+    "source": "client",
+    "destination": "processed",
+    "directory": "cmslite_gdx",
+    "doc": "gov_assets.*",
+    "dbtable": "cmslite.asset_downloads",
+    "column_count": 10,
+    "columns": [
+      "ip",
+      "id",
+      "uid",
+      "date/timestamp",
+      "request_string",
+      "status_code",
+      "return_size",
+      "referrer",
+      "user_agent_http_request_header",
+      "proxy"
+    ],
+    "drop_columns": [
+      "id",
+      "uid"
+    ],
+    "dtype_dic_strings": [
+      "Source Translated Port"
+    ],
+    "delim": "|",
+    "truncate": false,
+    "dateformat": [
+      {
+        "field": "date/timestamp",
+        "format": "%d/%b/%Y:%H:%M:%S -0700"
+      }
+    ],
+    "replace": [
+      {
+        "field": "request_string",
+        "old": "GET ",
+        "new": ""
+      },
+      {
+        "field": "request_string",
+        "old": " HTTP/1.0",
+        "new": ""
+      }
+    ],
+    "global_regex": {
+      "regex": {
+        "match1": { 
+          "match": "^(.*) (.*) (.*) \\[(.*)\\] \\\"(.*)\\\" (.*) (.*) \\\"(.*)\\\" \\\"(.*)\\\" (.*)$", 
+          "replace": "\\1|\\2|\\3|\\4|\\5|\\6|\\7|\\8|\\9|\\10"
+        },
+        "match2": {
+          "match": "^(.*) (.*) (.*) \\[(.*)\\] \\\"(.*)\\\" (.*) (.*) \\\"(.*)\\\" \\\"(.*)\\\"$", 
+          "replace": "\\1|\\2|\\3|\\4|\\5|\\6|\\7|\\8|\\9|"
+        }
+      },
```

Comment:
I would assign `"regex"` as an array instead of as an object containing members that are themselves objects. We know that such an array would contain only match/replace object pairs, so each one doesn't need its own name.

As a result this would become:
```
"regex": [
  { 
    "match": "^(.*) (.*) (.*) \\[(.*)\\] \\\"(.*)\\\" (.*) (.*) \\\"(.*)\\\" \\\"(.*)\\\" (.*)$", 
    "replace": "\\1|\\2|\\3|\\4|\\5|\\6|\\7|\\8|\\9|\\10"
  },
  {
    "match": "^(.*) (.*) (.*) \\[(.*)\\] \\\"(.*)\\\" (.*) (.*) \\\"(.*)\\\" \\\"(.*)\\\"$", 
    "replace": "\\1|\\2|\\3|\\4|\\5|\\6|\\7|\\8|\\9|"
  }
]
```

Reply:
Looks like you addressed this in the latest commit; I'll resolve this.